# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mdadm/dietlibc.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./config.c.orig	2007-07-18 21:10:32.000000000 -0400
+++ ./config.c	2007-07-18 21:10:45.000000000 -0400
@@ -29,7 +29,7 @@
 
 #include	"mdadm.h"
 #include	"dlink.h"
-#include	<sys/dir.h>
+#include	<dirent.h>
 #include	<glob.h>
 #include	<fnmatch.h>
 #include	<ctype.h>
--- ./Monitor.c.orig	2007-07-18 21:17:39.000000000 -0400
+++ ./Monitor.c	2007-07-18 21:17:57.000000000 -0400
@@ -34,6 +34,7 @@
 #include	<sys/signal.h>
 #include	<values.h>
 #include	<syslog.h>
+#include	<limits.h>
 
 static void alert(char *event, char *dev, char *disc, char *mailaddr, char *mailfrom,
 		  char *cmd, int dosyslog);
